projects/2DoBox-Pivot.md

```diff
@@ -0,0 +1,160 @@
+# 2DoBox Pivot
+
+Do you remember that project Ideabox?
+
+We thought they were pretty great, but, I think it's time to pivot those projects over to something we like to call 2DoBox.
+
+You will be inheriting someone's previous IdeaBox and pivoting it over to 2DoBox. Your projects will utilize webpack and it's built in node server. You will also keep accessibility in mind as we will be grading you based on how accessible your application is. You will also implement selenium webdriver to test the DOM.
+
+# Phase 1
+
+### Adding a new ToDo
+
+On the application’s main page, a user should:
+
+See two text boxes for entering the “Title” and “Task” for a new TODO, and a “Save” button for committing that TODO.
+When a user clicks “Save”:
+
+A new TODO with the provided title and body should appear in the TODO list.
+The text fields should be cleared and ready to accept a new TODO.
+The page should not reload.
+The TODO should be persisted. It should still be present upon reloading the page
+
+
+### Deleting an existing TODO
+
+When viewing the TODO list:
+
+Each TODO in the list should have a link or button to “Delete” (or 𝗫).
+Upon clicking “Delete”, the appropriate TODO should be removed from the list.
+The page should not reload when an idea is deleted.
+The TODO should be removed from localStorage. It should not re-appear on next page load.
+
+
+### Editing an existing TODO
+
+When a user clicks the title or task of a TODO in the list, that text should become an editable text field, pre-populated with the existing  TODO title or task.
+The user should be able to “commit” their changes by pressing “Enter/Return” or by clicking outside of the text field.
+If the user reloads the page, their edits will be reflected.
+
+
+# Phase 2
+
+### Accessibility
+
+Your web application should pass `aXe-core` tests and should be `tab index` accessible.
+
+### Importance
+
+Each  TODO should be given a level of importance. As a user I should be able to change the level of importance by up-voting or down-voting that specific  TODO. Each  TODO should start with a level of `Normal`.
+
+Levels of Importance are as follows
+
+1) Critical
+
+2) High
+
+3) Normal
+
+4) Low
+
+5) None
+
+The change of importance should persist after a page refresh
+
+### Bookmarking
+
+There should be a `bookmark` button for each  TODO. As a user if I click the `bookmark` button I should see a visual change for the specific  TODO I've bookmarked.
+
+If I refresh the page the book mark should persist.
```

Comment:
What is the purpose of the bookmark other than the visual change? Should they be able to see only the bookmarked to dos through a sort or something like that?